packages/kit/src/core/build/build_client.js

```diff
@@ -0,0 +1,119 @@
+import fs from 'fs';
+import path from 'path';
+import { svelte } from '@sveltejs/vite-plugin-svelte';
+import { deep_merge } from '../../utils/object.js';
+import { print_config_conflicts } from '../config/index.js';
+import { create_app } from '../create_app/index.js';
+import { copy_assets, posixify } from '../utils.js';
+import { create_build, find_deps } from './utils.js';
+
+/**
+ * @param {{
+ *   cwd: string;
+ *   assets_base: string;
+ *   config: import('types/config').ValidatedConfig
+ *   manifest_data: import('types/internal').ManifestData
+ *   build_dir: string;
+ *   output_dir: string;
+ *   client_entry_file: string;
+ *   service_worker_entry_file: string | null;
+ * }} options
+ */
+export async function build_client({
+	cwd,
+	assets_base,
+	config,
+	manifest_data,
+	build_dir,
+	output_dir,
+	client_entry_file
+}) {
+	create_app({
+		manifest_data,
+		output: build_dir,
+		cwd
+	});
+
+	copy_assets(build_dir);
+
+	process.env.VITE_SVELTEKIT_AMP = config.kit.amp ? 'true' : '';
```

Comment:
I don't see this used anywhere

Reply:
If it's meant to be exposed to user apps, it's not in the docs

Reply:
it's used here https://github.com/sveltejs/kit/blob/1ec368a9b90d65f13a818ff348187a7160544ea7/packages/kit/src/runtime/app/env.js#L16